catalog.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Гостинницы</title>
+  </head>
+  <body>
+    <header class="page-header">
+        <nav class="navigation">
+          <ul class="navigation-list">
+           <a class="navigation-logo" href="index.html">
+            <img class="logo" src="img/logo.svg" alt="logo Седона">
+           </a>
+            <li class="navigation-item">
+              <a class="navigation-link" href="main_page.html" >Главная</a>
```

Comment:
Конкретно эта ссылка отличается от остальных нижним подчеркиванием. Это значит, что она сейчас активная, это можно обозначить классом
```suggestion
              <a class="navigation-link navigation-link-active" href="main_page.html">Главная</a>
```